packages/core/core/src/TransformerRunner.js

```diff
@@ -209,7 +209,7 @@ export default class TransformerRunner {
         !transformer.canReuseAST(input.ast, this.options)) &&
       previousGenerate
     ) {
-      let output = await previousGenerate(input);
+      let output = await previousGenerate(new MutableAsset(input));
```

Comment:
How were the types not catching these before?

Reply:
not entirely certain, but it looks like their interfaces were compatible before. With `addURLDependency` that's no longer the case.